packages/account/src/Styles/account.scss

```diff
@@ -2210,7 +2216,7 @@ $MIN_HEIGHT_FLOATING: calc(
         }
     }
 
-    .deactivate-account-error {
+    .closing-account-error {
```

Comment:
These classnames are not changed in corresponding files -`packages/account/src/Sections/Security/ClosingAccount/account-has-balance.jsx`. could you pls check?